root/edit/details/AddRelationshipType.js

```diff
@@ -0,0 +1,169 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import IntentionallyRawIcon from '../components/IntentionallyRawIcon';
+import Cardinality from '../../static/scripts/common/components/Cardinality';
+import OrderableDirection
+  from '../../static/scripts/common/components/OrderableDirection';
+import {ENTITY_NAMES} from '../../static/scripts/common/constants';
+import expand2react from '../../static/scripts/common/i18n/expand2react';
+
+type AddRelationshipTypeEditT = {
+  ...EditT,
+  +display_data: {
+    +attributes: $ReadOnlyArray<{
+      ...LinkTypeAttrTypeT,
+      +typeName: string,
+    }>,
+    +child_order: number,
+    +description: string | null,
+    +documentation: string | null,
+    +entity0_cardinality?: number,
+    +entity0_type: CoreEntityTypeT,
+    +entity1_cardinality?: number,
+    +entity1_type: CoreEntityTypeT,
+    +link_phrase: string,
+    +long_link_phrase: string,
+    +name: string,
+    +orderable_direction?: number,
+    +reverse_link_phrase: string,
+  },
+};
+
+type Props = {
+  +edit: AddRelationshipTypeEditT,
+};
+
+const AddRelationshipType = ({edit}: Props): React.Element<'table'> => {
+  const display = edit.display_data;
+  const entity0Type = ENTITY_NAMES[display.entity0_type]();
+  const entity1Type = ENTITY_NAMES[display.entity1_type]();
+  const rawIconSection = (
+    <>
+      {' '}
+      <IntentionallyRawIcon />
+    </>
+  );
+
+  return (
+    <table className="details add-relationship-type">
+      <tr>
+        <th>{addColonText(l('Name'))}</th>
+        <td>
+          {display.name}
+          {rawIconSection}
+        </td>
+      </tr>
+
+      <tr>
+        <th>{addColonText(l('Description'))}</th>
+        <td>
+          {nonEmpty(display.description)
+            ? (
+              <>
+                {display.description}
+                {rawIconSection}
+              </>
+            ) : lp('(none)', 'description')}
+          {}
+        </td>
+      </tr>
+
+      <tr>
+        <th>{l('entity0 type:')}</th>
+        <td>{entity0Type}</td>
+      </tr>
+
+      <tr>
+        <th>{l('entity1 type:')}</th>
+        <td>{entity1Type}</td>
+      </tr>
+
+      <tr>
+        <th>{l('Link phrase:')}</th>
+        <td>
+          {display.link_phrase}
+          {rawIconSection}
+        </td>
+      </tr>
+
+      <tr>
+        <th>{l('Reverse link phrase:')}</th>
+        <td>
+          {display.reverse_link_phrase}
+          {rawIconSection}
+        </td>
+      </tr>
+
+      <tr>
+        <th>{l('Long link phrase:')}</th>
+        <td>
+          {display.long_link_phrase}
+          {rawIconSection}
+        </td>
+      </tr>
+
+      <tr>
+        <th>{l('entity0 cardinality:')}</th>
```

Comment:
Same as above, even though that localization message preexisted:

```suggestion
        <th>{addColonText(l('Cardinality of {entity0}'))}</th>
```

If changed here, it should be changed in these other files too:
- `root/edit/details/edit_relationship_type.tt`
- `root/relationship/linktype/RelationshipTypeIndex.js`
- `root/relationship/linktype/RelationshipTypePairTree.js`
- `root/relationship/linktype/form.tt`